person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    @id = rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services
```

Comment:
- [ ] **[OPTIONAL]** It is a Ruby convention to include question mark (?) in methods of this nature that return `true` or `false` boolean values. Just like the `of_age?`, I would recommend you change this method to this:

  ```
  def can_use_services?
    ...
  ```

  You can refer to [this Stackoverflow answer](https://stackoverflow.com/a/1345854) and the link it contains for a better understanding